### PR TITLE
Reverse the sub Array

### LINKED ISSUE #55: Reverse the sub Array
Given an array arr[] of positive integers of size N. Reverse every sub-array group of size K.

Note: If at any instance, there are no more subarrays of size greater than or equal to K, then reverse the last subarray (irrespective of its size).

### PR DESCRIPTION
**Pull request by neetu1123**
Reverse the sub array of group in .ts file 

>closes #55

### CODE REVIEW

**Comment by neetu1123** [comment]
close #48 reverse the group of sub array

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (neerajnanvani) [inline index.html]: delete this file

File: script.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+let arr=[1,2,3,4,5,6]
+        let k=4
+        let n=5
+        function check(arr,k,n){
+            let str=''
+            for(let i=0;i<=k;i+=3){
+             str =arr[i]
```
Comment (neerajnanvani) [inline script.js]: delete this file

File: script.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        let index:number=4
+        
+        //function of reverseArray
+        const reverseArr= (arr:( number)[],index:number) => {
+            let newArr:number
+            for(let i=0;i<=index;i+=3){
+            newArr =arr[+i]
+             arr[i]=arr[i+2]
+             arr[i+2]=newArr
+             console.log(arr)
```
Comment (neerajnanvani) [inline script.ts]: remove console

**Review by neerajnanvani — CHANGES_REQUESTED** [review]

**Comment by neetu1123** [comment]
updated close #48

**Comment by neetu1123** [comment]
i had closed by mistake , now reopened it

File: script.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+            let newArr:number
+            for(let i=0;i<=index;i+=3){
+            newArr =arr[+i]
+             arr[i]=arr[i+2]
+             arr[i+2]=newArr
+             
+            }
+         return arr
+        }
+        reverseArr(arr,index) // return reverse sub array
```
Comment (neerajnanvani) [inline script.ts]: this answer doesn't give me correct outputs in different arrays and different key

**Review by neerajnanvani — CHANGES_REQUESTED** [review]

File: script.ts [hunk 5]
```diff
@@ -1,16 +1,22 @@
...
-        const reverseArr= (arr:( number)[],index:number) => {
-            let newArr:number
-            for(let i=0;i<=index;i+=3){
-            newArr =arr[+i]
-             arr[i]=arr[i+2]
-             arr[i+2]=newArr
-             
-            }
-         return arr
+       const reverseArr= (arr,index) => {
```
Comment (neerajnanvani) [inline script.ts]: give types to arr and index parameters

File: script.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+        let index:number=3
+        
+        //function of reverseArray
+       const reverseArr= (arr:number[],index:number) => {
+        for (let i = 0; i < arr.length; i += index) {
+        let begin = i;
+        let end = Math.min(i + index - 1, arr.length - 1);
+        let temp;
+
+        while (start < end) {
```
Comment (neerajnanvani) [inline script.ts]: start number is undefined

**Comment by neetu1123** [comment]
closed

**Pull request closed without merging**